### DISCUSSION TITLE
💚Unlocker for sites like (inc42 and the Economic Times) ❤️

### DISCUSSION THREAD

**Discussion by AkshayPratapSingh09** [community/community#74692] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I searched for some _**unlockers on the internet**_ and didn't find one so I decided to make one, I want to make an **Unlocker** for sites like these to make them accessible to the underprivileged. Can anybody help me to come up with a approach and help me with the ideation phase, rest  we can discuss and contribute to this cause together .PS>::

Sites I want  to decode:
The Economic Times
INC42 
and many more

**Comment by ghost** [comment]
hacker hai bhai hacker hai bhaishab isko bolte hai hacking

**Comment by imwule** [comment]
cool !

**Comment by Akash1134** [comment]
Hi @AkshayPratapSingh09 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
